deps/libcdio.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "libcdio",
+    "cleanup": [
+        "/share/info"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://ftp.gnu.org/gnu/libcdio/libcdio-0.94.tar.gz",
+            "sha256": "96e2c903f866ae96f9f5b9048fa32db0921464a2286f5b586c0f02699710025a"
+        }
+    ]
+}
```

Comment:
Use latest version from libcdio, it's 2.0.0 Source: https://git.savannah.gnu.org/cgit/libcdio.git/
To get it compilable add this configs: `--without-cd-drive`, `--without-cd-info`, `--without-cdda-player`, `--without-cd-read`, `--without-iso-info` and `--without-iso-read` It works for the platform version 18.08.

Reply:
I found this, and that works great. thanks.